doc/source/f2py/windows/index.rst

```diff
@@ -0,0 +1,95 @@
+.. _f2py-windows:
+
+=================
+F2PY and Windows
+=================
+
+.. note::
+
+	F2PY support for Windows is not at par with Linux support, and 
+	OS specific flags can be seen via ``python -m numpy.f2py``
+
+Broadly speaking there are two issues working with F2PY on Windows, the lack of actively
+developed FOSS Fortran compilers, and the linking issues related to the C
+runtime library for building Python-C extensions.
+
+The focus of this chapter is to establish a guideline for developing and
```

Comment:
```suggestion
The focus of this page is to establish a guideline for developing and
```

Reply:
Since this is the index page linking to different pages, I thought perhaps chapter might be more appropriate along the lines of [other parts of the documentation](https://numpy.org/doc/stable/user/c-info.how-to-extend.html?highlight=chapter).

I'm not sure if those are deprecated holdovers from the NumPy book though.

Reply:
I think they are, maybe section or document would be more appropriate?

Reply:
Yup section sounds good, might be worth changing all instances of chapter to section elsewhere.